xml/System.Text.RegularExpressions/RegexOptions.xml

```diff
@@ -117,7 +117,7 @@
       </ReturnValue>
       <MemberValue>8</MemberValue>
       <Docs>
-        <summary>Specifies that the regular expression is compiled to an assembly. This yields faster execution but increases startup time. This value should not be assigned to the <see cref="P:System.Text.RegularExpressions.RegexCompilationInfo.Options" /> property when calling the <see cref="M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)" /> method. For more information, see the "Compiled Regular Expressions" section in the <see href="/dotnet/standard/base-types/regular-expression-options">Regular Expression Options</see> article.</summary>
+        <summary>Specifies that the regular expression will compile IL code on the fly, not compile time, runtime to pass string for that specific expression. This value should not be assigned to the <see cref="P:System.Text.RegularExpressions.RegexCompilationInfo.Options" /> property when calling the <see cref="M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)" /> method. For more information, see the "Compiled Regular Expressions" section in the <see href="/dotnet/standard/base-types/regular-expression-options">Regular Expression Options</see> article.</summary>
```

Comment:
I'm not sure I understand this part:

> on the fly, not compile time, runtime to pass string for that specific expression

1. I'm not sure "runtime" makes sense in this sentence.
2. "to pass string for that specific expression". I don't think that is grammatically correct.

Reply:
> I'm not sure I understand this part:
> 
> > on the fly, not compile time, runtime to pass string for that specific expression
> 
> 1. I'm not sure "runtime" makes sense in this sentence.
> 2. "to pass string for that specific expression". I don't think that is grammatically correct.

How about this? - "This string will be passed runtime"

Reply:
> How about this? - "This string will be passed runtime"

I still don't understand what it means.

Reply:
> > How about this? - "This string will be passed runtime"
> 
> I still don't understand what it means.

This may clarify that that this expression will be executed at runtime anyway.